src/test/java/spoon/test/position/PositionTest.java

```diff
@@ -126,6 +125,10 @@ public void testPositionParameterTypeReference() throws Exception {
 		CtTypeReference<?> field3Type =  foo.getField("field3").getType();
 		//this probably points to an bug in JDT. But we have no workaround in Spoon, which handles spaces and comments too
 		assertEquals("List<T // */ >\n\t/*// */>", contentAtPosition(classContent, field3Type.getPosition()));
+
+		CtTypeReference<?> field4Type =  foo.getField("field4").getType();
+		//this probably points to an bug in JDT. But we have no workaround in Spoon, which handles spaces and comments too
```

Comment:
Did you copy the comment on purpose?

Reply:
thanks for pointing to that. It was not intentional